src/assets/articleAssets/demo2.scss

```diff
@@ -497,5 +499,41 @@ a:hover {
   margin-left: 2px;
   font-size: 30px;
 }  
-  
-  
+
+.article-container {
+  // column-count: 1;
+  width: 100%;
+  // padding-left: 40px;
+  // padding-right: 40px !important;
+  // margin: 0 auto 0 0;
```

Comment:
Remove all commented code in all files, ensure to check through all files before you push any commented code, also add a new line at the end of the file. Rename this scss file to a more descriptive file name

Reply:
I have removed the commented out code and renamed the demo2.scss file to articlepage.scss